src/lib/output/plugins/JavascriptIndexPlugin.ts

```diff
@@ -63,12 +64,15 @@ export class JavascriptIndexPlugin extends RendererComponent {
                 parent = undefined;
             }
 
-            const row: any = {
+            const row: IDocument = {
                 id: rows.length,
                 kind: reflection.kind,
                 name: reflection.name,
                 url: reflection.url,
-                classes: reflection.cssClasses,
+                classes: reflection.cssClasses ?? "",
```

Comment:
I'd like to leave this as is - again looking to avoid increasing the search.js file size.

Reply:
Oh, gotcha. I think I was getting some kind of build complaint somewhere, but between this and the errors I was getting from the dom stuff, maybe I have something configured wrong. I'll try to narrow it down tomorrow.

Reply:
I think it's because IDocument doesn't define this property as optional - it should be

Reply:
Makes sense. I'll touch that up, thank you.

Reply:
009a2442